neptune-export/src/main/java/com/amazonaws/services/neptune/export/ExportToS3NeptuneExportEventHandler.java

```diff
@@ -348,10 +350,13 @@ private void uploadExportFilesToS3(TransferManager transferManager, File directo
 
                 AmazonClientException amazonClientException = upload.waitForException();
 
+                // override AmazonClientException's isRetryable instead?
                 if (amazonClientException != null){
                     String errorMessage = amazonClientException.getMessage();
+                    // only retry on 5xx? or no-retry on 4xx?
+                    Matcher exMsgStatusCodeMatcher = Pattern.compile("Status Code: (5\\d+)").matcher(errorMessage);
```

Comment:
better to make static variable for compiled pattern once and reuse

Reply:
Updated